<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spēle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>481</width>
        <height>401</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>481</width>
        <height>401</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageStart">
       <widget class="QGroupBox" name="groupPlayer">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>241</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0</string>
        </property>
        <property name="title">
         <string>Pirmais spēlētājs</string>
        </property>
        <widget class="QRadioButton" name="radioUser">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Lietotājs</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioComputer">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>30</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Dators</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupAlgorithm">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>130</y>
          <width>241</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0</string>
        </property>
        <property name="title">
         <string>Algoritms</string>
        </property>
        <widget class="QRadioButton" name="radioMinimax">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Minimakss</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioAlfaBeta">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>30</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Alfa-Beta</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="btnStartGame">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>330</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Sākt spēli</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupLength">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>210</y>
          <width>241</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QSlider" name="sliderLength">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>40</y>
           <width>161</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimum">
          <number>15</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLabel" name="lblLength">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>-10</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Virknes garums 15</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblLength_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>15</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblLength_3">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>60</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>16</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblLength_4">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>60</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>17</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblLength_5">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>60</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>18</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblLength_6">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>60</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>19</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblLength_7">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>60</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>20</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="pageMain">
       <widget class="QLabel" name="lblWinner">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>110</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Uzvarētājs</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lblNumbers">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>110</y>
          <width>211</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lblBankNum">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblBank">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>60</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Banka</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblPointsNum">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>40</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblPoints">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>40</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Punkti</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnLeft">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>170</y>
          <width>31</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;-</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnRight">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>170</y>
          <width>31</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>-&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnRemove">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>170</y>
          <width>71</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Pieskaitīt</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDivide">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>200</y>
          <width>71</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Sadalīt</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnNewGame">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>350</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Jauna spēle</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblCurrentPlayer">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>40</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Spēlētajs</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
